pyradiosky/skymodel.py

```diff
@@ -1659,18 +1902,36 @@ def read_votable_catalog(
             spectral_type = "subband"
             spectral_index = None
 
-        stokes = np.zeros((4, len(flux_cols_use), len(astropy_table)), dtype=np.float)
+        col_units = []
         for index, col in enumerate(flux_cols_use):
-            stokes[0, index, :] = astropy_table[col].data.data
+            col_units.append(astropy_table[col].unit)
```

Comment:
Is it generally guaranteed that the astropy table read in will have units defined for each column? If not, we should make it clear in docs that this is expected.

Reply:
So the units are actually in the VOTable file and just get pulled into the astropy table object from there. We can add some docs, but it appears that units are usually included in VOTable files (from the handful I looked at.

Reply:
Okay! That's really convenient. It looks like units are not required for the [VOTable format](http://www.ivoa.net/documents/VOTable/20040811/REC-VOTable-1.1-20040811.html#ToC27), but the error on line 1920 should be raised if a user doesn't provide a VOTable with units.